lib/controller/matlab/WB_NODE_DIFFERENTIAL_WHEELS.m

```diff
@@ -1,2 +0,0 @@
-function value = WB_NODE_DIFFERENTIAL_WHEELS
-value = 32;
```

Comment:
Note that after deleting the `WB_NODE_DIFFERENTIAL_WHEELS` node, all the subsequent node values changes and the whole Matlab API should be updated and not just this file.
So you should regenerate the Matlab API files and commit all the changed ones.